firmware/configs/campzone2019_defconfig

```diff
@@ -35,7 +38,7 @@ CONFIG_FLASH_ENCRYPTION_ENABLED=
 #
 # Serial flasher config
 #
-CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
+CONFIG_ESPTOOLPY_PORT="`ls /dev/tty{USB0,.wchusbserial*,.SLAB_USBtoUART,ACMx} 2>/dev/null`"
```

Comment:
This only works on mac os, not on linux!

Reply:
This works on LTS Ubuntu and Fedora. We have another issue open for it, to figure out on what systems it doesn't. Since I build releases for this badge on Mac OS, I will keep it like this until we figure it out.